apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -50,77 +52,133 @@ export function InstructorCourseAdminInstances({
+                              data-placement="bottom"
+                              data-html="true"
+                              title="Latest Access Date"
+                              data-content="${PopoverEndDate()}"
+                              aria-label="Information about Latest Access Date"
+                            >
+                              <i class="far fa-question-circle" aria-hidden="true"></i>
+                            </button>
+                          </th>
+                          <th>Students</th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        ${courseInstances.map((row) => {
+                          return html`
+                            <tr>
+                              <td class="align-left">
+                                ${row.sync_errors
+                                  ? SyncProblemButton({
+                                      type: 'error',
+                                      output: row.sync_errors,
+                                    })
+                                  : row.sync_warnings
+                                    ? SyncProblemButton({
+                                        type: 'warning',
+                                        output: row.sync_warnings,
+                                      })
+                                    : ''}
+                                <a
+                                  href="${resLocals.plainUrlPrefix}/course_instance/${row.id}/instructor/instance_admin"
+                                  >${row.long_name}</a
+                                >
+                              </td>
+                              <td class="align-left">${row.short_name}</td>
+                              <td class="align-left">${row.formatted_start_date}</td>
+                              <td class="align-left">${row.formatted_end_date}</td>
+                              <td class="align-middle">${row.enrollment_count}</td>
+                            </tr>
+                          `;
+                        })}
+                      </tbody>
+                    </table>
+                  </div>
+                `
+              : html`
+                  <div class="my-4 card-body text-center" style="text-wrap: balance;">
+                    <p class="font-weight-bold">No course instances found.</p>
+                    <p class="mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
+                    </p>
+                    <p>
+                      Learn more in the
+                      <a
+                        href="https://prairielearn.readthedocs.io/en/latest/courseInstance/"
+                        target="_blank"
+                        >course instance documentation</a
+                      >.
```

Comment:
Based on my conversation with @mwest1066 about this project to improvement the onboarding experience, I understood the first goal was to remove the need to edit json files, or further, to remove the need from the user to understand the format of these json files. If that is the case, I don't think we should link to the documentation that explains infoCourseInstance.json. 

Is this just a placeholder for future better documentation that we will provide to users?

Reply:
Yep, I'm thinking that anything we put here is going to be a placeholder. Before we write documentation that talks about course instance config independently of JSON, we need to have creation/edit flows in the PL web UI. I figure at this point, some docs are better than none, and understanding how to edit JSON _will_ still be necessary until nice web editing UIs are in place.

Reply:
agree!